packages/server-node/src/base-impl/gmodel-storage.ts

```diff
@@ -72,4 +74,14 @@ export class GModelLoader implements ModelSourceLoader {
             throw new GLSPServerError(`Could not read & parse file contents of '${url}' as json`, error);
         }
     }
+
+    saveSourceModel(action: SaveModelAction): MaybePromise<void> {
+        try {
+            const data = this.modelSerializer.createSchema(this.modelState.root);
+            // eslint-disable-next-line no-null/no-null
+            writeFileSync(this.modelState.sourceUri!, JSON.stringify(data, null, 2));
```

Comment:
Instead of disabling the eslint rule you can just use `undefined` here instead of null
```suggestion
            writeFileSync(this.modelState.sourceUri!, JSON.stringify(data, undefined, 2));
```

Reply:
Ah right, yeah I just moved the code. I amended your change. Thanks!